engine/schema/src/main/resources/META-INF/db/schema-41100to41110-cleanup.sql

```diff
@@ -16,5 +16,5 @@
 -- under the License.
 
 --;
--- Schema upgrade cleanup from 4.11.0.0 to 4.12.0.0
+-- Schema upgrade from 4.11.0.0 to 4.11.1.0
```

Comment:
@DaanHoogland, Thanks for the PR. I think it would be more meaningful if we add the word "cleanup" here.

Reply:
will do